var/spack/repos/builtin/packages/hpgmg/package.py

```diff
@@ -78,7 +77,11 @@ def configure_args(self):
         if 'fv=none' in self.spec:
             args.append('--no-fv')
         else:
-            cflags.append(self.compiler.openmp_flag)
+            # Apple's Clang doesn't support OpenMP
+            if (self.spec.satisfies('threads=openmp') and not
+                (self.spec.satisfies('%clang') and
+                 str(self.spec.compiler.version).endswith('-apple'))):
+                cflags.append(self.compiler.openmp_flag)
```

Comment:
Some day Apple will enable OpenMP in default Clang on Mac. My understanding is that if you build Clang from source, you can have OpenMP on Mac now. I'd rather just make OpenMP non-default (that appears to have been done here) and then do what the user says.